lib/xdrgen/generators/csharp.rb

```diff
@@ -386,11 +386,11 @@ def encode_type(type, value)
         when AST::Typespecs::Int
           "stream.WriteInt(#{value})"
         when AST::Typespecs::UnsignedInt
-          "stream.WriteInt(#{value})"
+          "stream.WriteUInt(#{value})"
```

Comment:
Where is the `WriteUint` function defined? I can't find it in:

https://github.com/stellar/xdrgen/blob/9dacf24a73b3e73105e9272ccf333704dcc82333/lib/xdrgen/generators/java/XdrDataOutputStream.erb#L8-L9

Or in the docs for `java.io.DataOutputStream`:

https://docs.oracle.com/javase/7/docs/api/java/io/DataOutputStream.html

I'm very rusty on C# though, so I'm sure I'm missing something.

Reply:
It may be beneficial to build that logic into the generator so it is possible to use it to generate other XDR formats, but that is unrelated to this change and this PR.